biorxiv/article_distances/biorxiv_article_distances.py

```diff
@@ -0,0 +1,499 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Find published articles missing from bioRxiv
+
+# In[1]:
+
+
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import plotnine as p9
+from scipy.spatial.distance import cdist
+import tqdm
+
+
+# # Load Embeddings
+
+# ## bioRxiv
+
+# In[2]:
+
+
+biorxiv_journal_df = (
+    pd.read_csv(
+        "../journal_tracker/output/mapped_published_doi.tsv", 
+        sep="\t"
+    )
+    .groupby("doi")
+    .agg({
+        "document":"last",
+        "category":"first",
+        "journal":"first",
+        "doi":"last",
+        "published_doi":"first",  
+        "pmcid":"first", 
+        "pmcoa":"first"
+    })
+    .reset_index(drop=True)
+)
+biorxiv_journal_df.head()
+
+
+# In[3]:
+
+
+biorxiv_embed_df = (
```

Comment:
What is the difference between biorxiv_journal_mapped_df and boirxiv_embed_df?

Reply:
>What is the difference between biorxiv_journal_mapped_df and boirxiv_embed_df?

biorxiv_journal_mapped_df - dataframe with biorxiv preprints linked with their published versions
biorxiv_embed_df - dataframe that contains document embeddings for preprints